zerver/openapi/zulip.yaml

```diff
@@ -20027,6 +20033,45 @@ paths:
                       "new": {"direct_members": [10], "direct_subgroups": [11]},
                       "old": 15,
                     }
+                can_administer_channel_group:
+                  description: |
+                    The set of users who have permission to administer this channel
+                    expressed as an [update to a group-setting value][update-group-setting].
+
+                    Note that a user who is a member of the specified user group must
+                    also [have access](/help/channel-permissions) to the channel in
+                    order to administer the channel.
+
+                    Realm admins are allowed to administer a channel they have access to
+                    regardless of whether they are present in this group.
+
+                    Users in this group can edit channel name and description without
+                    subscribing to the channel, but they need to be subscribed to edit
+                    channel permissions and add users.
+
+                    **Changes**: New in Zulip 10.0 (feature level 322).
```

Comment:
These should always explain the previous state. "Previously, only realm administrators could administer channels."